modules/check_stop_condition.py

```diff
@@ -58,4 +58,4 @@ def check_stop_condition(start_time: float,
     if not result:
         print("Unable to upload rtl command to drone command sequence.")
 
-    return True
+    return True
```

Comment:
Nit: Add an extra line at the end of the file. If you see the red sign on a file, it means you forgot to add the extra line.

Reply:
Please also add lines for other files with the red sign.